R/formulas.R

```diff
@@ -195,55 +173,82 @@ prb <- function(assessed, sale_price, na.rm = FALSE) {
-#' calculates the difference (GINI of Assessed - GINI of Sale), and the
-#' Modified Kakwani Index calculates the ratio (GINI of Assessed / GINI of Sale).
+#' @description The Kakwani Index (KI) and the Modified Kakwani Index (MKI)
+#'   are Gini-based methods to measure vertical equity.
 #'
-#' KI < 0 is regressive
-#' KI = 0 is vertical equity
-#' KI > 0 is progressive
+#'   These methods first order properties by sale price (ascending), then
+#'   calculate the Gini coefficient for sale values and assessed values (while
+#'   remaining ordered by sale price). The Kakwani Index then
+#'   calculates the difference \code{(Gini of assessed - Gini of sale)}, and the
+#'   Modified Kakwani Index calculates the ratio
+#'   \code{(Gini of Assessed / Gini of Sale)}.
 #'
-#' MKI < 1 is regressive
-#' MKI = 1 is vertical equity
-#' MKI > 1 is progressive
+#'   For the Kakwani Index:
 #'
-#'   \href{https://researchexchange.iaao.org/jptaa/vol17/iss2/2/}{A Gini measure for vertical equity in property assessments}
+#'   - KI < 0 is regressive
+#'   - KI = 0 is vertical equity
+#'   - KI > 0 is progressive
 #'
-#' @inheritParams ki_mki_met
-#' @describeIn ki_mki returns either the MKI or KI of numeric indicators
-#' based on boolean indicator.
+#'   For the Modified Kakwani Index:
 #'
-#' @param na.rm Default FALSE. A boolean value indicating whether or not to
-#' remove NA values. If missing values are present but not removed the
-#' function will output NA.
-#' @param assessed vector or row of assessed values
-#' with same length as \code{sale_price}
-#' @param sale_price vector or row of sale values
-#' with same length as \code{assessed}
-#' @param mki Default TRUE. A boolean indicator to identify if the output
-#' is mki or ki. MKI is the preferred metric.
+#'   - MKI < 1 is regressive
+#'   - MKI = 1 is vertical equity
+#'   - MKI > 1 is progressive
 #'
-#' @order 1
+#' @references
+#'   Quintos, C. (2020). A Gini measure for vertical equity in property
+#'   assessments. Journal of Property Tax Assessment & Administration, 17(2).
+#'   Retrieved from \href{https://researchexchange.iaao.org/jptaa/vol17/iss2/2}{https://researchexchange.iaao.org/jptaa/vol17/iss2/2}.
+#'
+#'   Quintos, C. (2021). A Gini decomposition of the sources of inequality in
+#'   property assessments. Journal of Property Tax Assessment & Administration,
+#'   18(2). Retrieved from
+#'  \href{https://researchexchange.iaao.org/jptaa/vol18/iss2/6}{https://researchexchange.iaao.org/jptaa/vol18/iss2/6}
+#'
+#' @inheritParams prd
+#' @describeIn mki_ki Returns a numeric vector containing the KI of the
+#'   input vectors.
+#' @order 2
 #'
-#' @family formulas
 #' @examples
-#' # example code
-#' result <- ki_mki(ratios_sample$assessed, ratios_sample$sale_price, mki = TRUE)
 #'
+#' # Calculate KI
+#' ki(ratios_sample$assessed, ratios_sample$sale_price)
+#' @family formulas
 #' @export
-#'
-ki_mki <- function(assessed, sale_price, na.rm = FALSE, mki = TRUE) {
```

Comment:
I separated `ki()` and `mki()` into their own function since I figured having them combined would be confusing in the long run and also doesn't match the other functions in this package. They share the same docs so the actual function is extremely small for both.